app/javascript/src/Navigation/components/ContextSelector.jsx

```diff
@@ -1,26 +1,37 @@
-// TODO: @flow
-// TODO: test
```

Comment:
Same here, this PR is not adding tests

Reply:
They were added already: https://github.com/3scale/porta/blob/master/spec/javascripts/Navigation/ContextSelector.spec.jsx.